bib/cmd/bootc-image-builder/main.go

```diff
@@ -43,6 +43,29 @@ type BuildConfig struct {
 	Blueprint *blueprint.Blueprint `json:"blueprint,omitempty"`
 }
 
+// canChownInPath checks if the ownership of files can be set in a given path.
+func canChownInPath(path string) (bool, error) {
+	if info, err := os.Stat(path); err == nil {
+		if !info.IsDir() {
+			return false, fmt.Errorf("%s is not a directory", path)
+		}
+	} else {
+		return false, err
+	}
+	checkFile, err := os.CreateTemp(path, ".writecheck")
+	if err != nil {
+		return false, err
+	}
+	defer func() {
+		err := os.Remove(checkFile.Name())
```

Comment:
(super nitpick) - I like  the one line pattern for things like this, i.e. `if err := os.Remove(...); err != nil {` as it's slightly more compact and err is in the if scope only. But it's probably just personal preference :)

Reply:
I prefer it too.  Not sure how I missed it there.